reference/forms/types/dateinterval.rst

```diff
@@ -56,8 +56,7 @@ or an array (see `input`_).
 Basic Usage
 -----------
 
-This field type is highly configurable, but easy to use. The most important
-options are `input`_ and `widget`_.
+The most important options are `input`_ and `widget`_.
```

Comment:
i would keep "This field type is highly configurable." but without the ", but easy to use" part.